mslib/utils/config.py

```diff
@@ -464,19 +504,21 @@ def load_settings_qsettings(tag, default_settings=None, ignore_test=False):
     return default_settings
 
 
-def merge_data(options, json_file_data):
+def merge_data(dict_to_merge_new_dict_into, new_dict):
```

Comment:
I have renamed the parameters of the merge data function as it was a bit confusing. Please let me know what you think.

Reply:
the function could also be merge_dict? shorter parameter could be perhaps then given_dict, new_dict or just given, new.

Reply:
:)